AssignCardToSender/index.ts

```diff
@@ -0,0 +1,13 @@
+import { OctokitAction } from '../lib/OctokitAction';
+
+class AssignCardToSender extends OctokitAction {
```

Comment:
Related to git work: When renaming file,s you should stage them, so git can recognize rename for the commit history.

It will also enable to see what was the actual formatting change, if any.

Reply:
Yeah, I guess, I got a bit confused in my git commits with the various renamings and deletions, which led to git not recognizing it as a single renaming action.

Reply:
It might also have confused git that I ran the `prettier` formatter in addition to moving the files.